idioms-json-2.0-custom/campaign-v-actors.json

```diff
@@ -55,7 +60,34 @@
         },
         {
             "created": "2014-08-08T15:50:10.983Z",
-            "id": "relationship--3dcf59c3-30e3-4aa5-9c05-2cbffcee5922",
+            "id": "relationship--5dcfc21e-c88c-4a3b-b68a-9d207995b0c8",
+            "modified": "2014-08-08T15:50:10.983Z",
+            "relationship_type": "attributed-to",
+            "source_ref": "incident--229ab6ba-0eb2-415b-bdf2-079e6b42f51e",
```

Comment:
It appears the issue only happens with this test file...

Reply:
The incidents aren't present in the xml file - just the idrefs.

This is an example of dangling references.  All of the referenced objects do not have to be in the bundle.

I set flag for incidents to True for all testing.  See idioms-json-2.0-custom/incident-malware.json

Do you think it should be off, at least for the non-custom 2.0 test run?

Reply:
Ahh I see! We can leave as-is then. I was thinking the code was not reaching the line that created such Incident.